File: Program/LawCreator/bin/test/lawcreator/utils/MapperTest.js
Clazz.declarePackage ("test.lawcreator.utils");
Clazz.load (null, "test.lawcreator.utils.MapperTest", ["pl.wroc.pwr.student.lawcreator.utils.Mapper"], function () {
c$ = Clazz.declareType (test.lawcreator.utils, "MapperTest");
Clazz.defineMethod (c$, "mappingStringsTest", 
function () {
org.junit.Assert.assertEquals ("uchwala", pl.wroc.pwr.student.lawcreator.utils.Mapper.map ("Uchwała"));
});
});

File: Program/LawCreator/test/test/lawcreator/utils/MapperTest.java
package test.lawcreator.utils;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

import pl.wroc.pwr.student.lawcreator.utils.Mapper;

public class MapperTest {

	@Test
	public void mappingStringsTest() {
		assertEquals("uchwala", Mapper.map("Uchwa³a"));
	}

}
